### ISSUE TITLE
Get Heroku CI to play nice with headless chromedriver

### ISSUE THREAD

**Issue by veganstraightedge** [crimethinc/website#450]
Right now, Travis CI passes, but Heroku CI fails. Heroku fails because it doesn't know what to do about that headless chromedriver. Can you see if you can get this fixed up on a test Heroku app, @astronaut-wannabe so that our build goes back to green in Heroku CI land, please?

```
-----> Running Ruby buildpack tests...
-----> Running test: bin/rspec
       .....................................................F......................................................................................................................................
       Failures:
       1) Setting and changing an articles published_at date Using 'PUBLISH NOW' feature
       Failure/Error: visit '/signin'
       Selenium::WebDriver::Error::WebDriverError:
       Unable to find chromedriver. Please download the server from http://chromedriver.storage.googleapis.com/index.html and place it somewhere on your PATH. More info at https://github.com/SeleniumHQ/selenium/wiki/ChromeDriver.
       # ./spec/spec_helper.rb:25:in `login_user'
       # ./spec/features/article_datetime_settings_spec.rb:81:in `block (2 levels) in <top (required)>'
       Finished in 31.91 seconds (files took 5.17 seconds to load)
       188 examples, 1 failure
       Failed examples:
       rspec ./spec/features/article_datetime_settings_spec.rb:75 # Setting and changing an articles published_at date Using 'PUBLISH NOW' feature
       Coverage report generated for RSpec to /app/coverage. 5411 / 12970 LOC (41.72%) covered.
-----> Ruby buildpack tests failed with exit status 1
```

**Comment by astronaut-wannabe** [comment]
sure thing, will fix this tomorrow

**Comment by veganstraightedge** [comment]
Thanks! 

Also, I've invited you to be a member of the repo, so that issues can be assigned to you proper and PRs can be branches on this repo.

**Comment by astronaut-wannabe** [comment]
👍

**Comment by astronaut-wannabe** [comment]
resolved in #452
